lib/grizzly/inclusion.ex

```diff
@@ -551,7 +565,8 @@ defmodule Grizzly.Inclusion do
     opts
     |> Keyword.put_new(:timeout, 60_000)
     |> Keyword.put_new(:client, self())
-    |> Keyword.put_new(:s2_keys, [])
+
+    #    |> Keyword.put_new(:s2_keys, [])
```

Comment:
I don't think we want that commented out?

Reply:
fixed